scripts/build-dist.sh

```diff
@@ -15,7 +15,7 @@ tar -xzf pack.tgz --strip 1
 rm -rf pack.tgz
 # Change this to "yarn install --production" once #1115 is fixed
 npm install --production
-rm -rf node_modules/*/test node_modules/*/dist
```

Comment:
Had to remove the blind `dist` folder removal rule because `rx` distribute their source code in a folder called `dist`.